doc/sphinx/reference/onedim/discretization.md

```diff
@@ -1,5 +1,361 @@
 ```
+
+The 1D solver sweeps over all points in a domain and computes a residual for each grid
+point and each equation. The solver attempts to find the combination of the variables
+that minimizes the residual vector.
+
+The solution vector $x$ is a vector of all the variables in the domain, at all grid
+points.
+
+$$
+\mathbf{x} =
+\begin{pmatrix}
+u_0 \\
+V_0 \\
+T_0 \\
+\Lambda_0 \\
+Y_{1,0} \\
+\vdots \\
+u_1 \\
+V_1 \\
+\vdots \\
+Y_{m,N}
+\end{pmatrix}
+$$
+
+This residual vector has entries in a form like what is shown below.
+
+$$
+\mathbf{F}(\mathbf{x}) =
+\begin{pmatrix}
+F_{u,0}(\mathbf{x}) \\
+F_{V,0}(\mathbf{x}) \\
+F_{T,0}(\mathbf{x}) \\
+F_{\Lambda,0}(\mathbf{x}) \\
+F_{Y_1,0}(\mathbf{x}) \\
+\vdots \\
+F_{u,1}(\mathbf{x}) \\
+F_{V,1}(\mathbf{x}) \\
+\vdots \\
+F_{Y_m,N}(\mathbf{x})
+\end{pmatrix}
+$$
+
+This is a vector-value function of the solution vector.
+
+For each governing equation, there is a default boundary condition applied at the left
+and right boundaries. During the sweep, the final evaluations are done for the domain
+boundary points, which allows for the default residual to be augmented or overwritten
+by a boundary condition that is different from the default boundary condition.
+
+## Continuity Equation
+
+The continuity equation is:
+
+$$
+ \frac{\partial (\rho u)}{ \partial z} + 2\rho V = 0
+$$
+
+### Discretization
+
+Because the continuity equation is first order and the boundary condition is applied at
+the right boundary in the 1D domain, the first derivative approximation uses a
+one-sided form which uses a stencil that pulls from points to right of a grid point.
+For consistency, the radial momentum term, ($2\rho V$), also uses the same points and
+averages the value between two nodes.
+
+The discretized equation in residual form (all terms moved to one side) at the interior
+points in the domain is given below.
+
+$$
+F_{u,j} = \frac{\rho_j u_j - \rho_{j+1} u_{j+1}}{z_{j+1} - z_j} +
+  2 \left( \frac{\rho_j V_j + \rho_{j+1} V_{j+1}}{2} \right)
+$$
+
+### Boundary Conditions
+
+At the right boundary, the default boundary condition is a zero velocity condition,
+which is representative of a stagnation surface.
+
+At the right boundary ($j=N$):
+
+$$
+F_{u,j} = \rho_j u_j - \dot m
+$$
+
+Expressing the boundary residual in this form will drive the Newton root finding
+algorithm to find the value of $ \rho_j u_j $ that minimizes the difference between it
+and the imposed (from a user-defined boundary mass flow rate specification, for
+example) boundary condition.
+
+There is no imposed boundary condition at the left boundary because only one boundary
+condition can be enforced for a first-order differential equation. As such, the
+residual equation at the left boundary is the same as the interior points.
+
+At the left boundary ($j=0$):
+
+$$
+F_{u,j} = \frac{\rho_j u_j - \rho_{j+1} u_{j+1}}{z_{j+1} - z_j} +
+  2 \left( \frac{\rho_j V_j + \rho_{j+1} V_{j+1}}{2} \right)
+$$
```

Comment:
Is this a default boundary condition though? Do we want to document all of those conditions as well here? I had originally planned to just do the ones that the flow domain imposed.

Reply:
Maybe it would be best to reserve that update for a separate PR, but I think the boundary conditions are implemented in the code, with the equations split between the flow domain and the connector, should be regarded as an implementation detail -- it's not really something that affects the mathematical formulation that's presented here.

Reply:
That makes sense. So essentially just discuss the primary discretization for the boundary conditions and flow.  I think as long as the actual header documentation discusses the existence of a default set of boundary conditions, that would be good. From my personal experience, it was a bit confusing to figure out what was going on  between the domain and the BC objects. Where a BC object might just add something to the residual vector, but it was clear what else was in that entry.